FanartHandler/FanartSelectedOther.cs

```diff
@@ -304,6 +321,22 @@ public void RefreshSelected(RefreshWorker rw, System.ComponentModel.DoWorkEventA
               Utils.ContainsID(WindowsUsingFanartSelectedAwardMovie))
           {
             IsSelectedVideo = true;
+            if (Utils.iActiveWindow == 87266 || // Trakt Trending Movies
+                Utils.iActiveWindow == 87700 || // Trakt Calendar Movies
+                Utils.iActiveWindow == 87101 || // Trakt Popular Movies
+                Utils.iActiveWindow == 87263 || // Trakt Recommendations Movies
+                Utils.iActiveWindow == 87605 || // Trakt Anticipated Movies
+                Utils.iActiveWindow == 87607 || // Trakt Box Office
+                Utils.iActiveWindow == 87265 || // Trakt Trending Series
+                Utils.iActiveWindow == 87102 || // Trakt Popular Series
+                Utils.iActiveWindow == 87262 || // Trakt Recommendations Series
+                Utils.iActiveWindow == 87259 || // Trakt Calendar Series
+                Utils.iActiveWindow == 87606 || // Trakt Anticipated Series
+                Utils.iActiveWindow == 87281 || // Trakt Series Seasons
+                Utils.iActiveWindow == 87282) // Trakt Series Episodes

```

Comment:
I may have forgotten the logic of how FH works, but in this way we remove the display of all selected images in these windows, but judging by the names, it is possible to display Clear Art, etc. for Movies, Series, etc. there.

Reply:
Currently I am adding this to xml files to get the FH to work on them:
`<!-- #fanarthandler.movie.genres.selected -->`

But instead we need a value like this for clearart:
`Utils.ContainsID(WindowsUsingFanartSelectedClearArt)`

The if statement above this code would then be updated to:
```
if (Utils.ContainsID(WindowsUsingFanartSelectedGenreMovie) || 
  Utils.ContainsID(WindowsUsingFanartSelectedStudioMovie) ||
  Utils.ContainsID(WindowsUsingFanartSelectedAwardMovie) ||
  Utils.ContainsID(WindowsUsingFanartSelectedClearArt))
```

Reply:
> But instead we need a value like this for clearart:
We need to add it (**WindowsUsingFanartSelectedClearArtMivie**) by analogy
https://github.com/andrewjswan/mediaportal-fanart-handler/blob/da7eec95eadc06967ea0fc907873679e4c15d612/FanartHandler/FanartSelectedOther.cs#L64

Reply:
Sure I can try ClearArtMusic and see if it works